.github/workflows/test.yml

```diff
@@ -4,12 +4,12 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@f90c7b395dac7c5a277c1a6d93d5057c1cddb74e # v2
+      - uses: actions/checkout@v2.3.4
```

Comment:
Could you revert this change? We have a security policy where we fix GitHub actions to SHAs instead of tags.

Reply:
reverted